### DISCUSSION TITLE
Reorder favorite repos

### DISCUSSION THREAD

**Discussion by sethfri** [community/community#3797] [Mobile] [answered]
Currently, Favorite Repos on the iOS app are sorted in alphabetical order with no way to reorder them. Some repos are more of my favorite than others 😊 It'd be great to provide a way to reorder this list at will.

Thanks!

**Comment by eliperkins** [comment, marked as answer]
Thanks for this feedback @sethfri! Reordering favorite repositories is on our radar as well. I've added your notes about this to our internal tracking issue for this. Stay tuned!

**Reply by sethfri** [reply]
It looks like this has been implemented, so marking this as an answer
